ports/mimxrt/boards/MIMXRT1064_EVK/mpconfigboard.mk

```diff
@@ -0,0 +1,9 @@
+MCU_SERIES = MIMXRT1064
+MCU_VARIANT = MIMXRT1064DVL6A
+
+JLINK_PATH = /media/RT1064-EVK/
+
+CFLAGS += -DBOARD_FLASH_SIZE=0x400000
```

Comment:
other boards have this as a `#define` in the `mpconfigboard.h` file; can the same be done for this board?

Reply:
The macro *BOARD_FLASH_SIZE* is used in this [file](https://github.com/hathach/nxp_driver/blob/b618cb1d521cc9e133bdcd0fca154dee2d925dfe/sdk/devices/MIMXRT1064/xip/fsl_flexspi_nor_boot.h), and this file only includes "board.h". I think only the definition in "board.h" is useful and the definition of "BOARD_FLASH_SIZE" in mpconfigboard.h is useless. But in currently nxp_driver submodule, BOARD_FLASH_SIZE is not defined in [board.h](https://github.com/hathach/nxp_driver/blob/b618cb1d521cc9e133bdcd0fca154dee2d925dfe/sdk/devices/MIMXRT1064/project_template/board.h). So for this boards, I use this way to define BOARD_FLASH_SIZE macro.

Reply:
ok, sounds good then!